app/services/geoapify_service.rb

```diff
@@ -0,0 +1,54 @@
+class GeoapifyService
+  def self.get_places(city_info, categories = nil, offset = 0, radius_mult = 1)
+    categories ||= default_categories
```

Comment:
Is there a reason you chose to do this rather than having categories = default_categories on line 2? It would set the default categories accordingly and change them if something else is supplied.

Reply:
Yeah, I couldn't quite figure out why, but it didn't play nice with calling the array method from the initializing arguments. I think it might've had something to do with nil being passed in to the call from the previous method? Idk. I have a lot of redundant default variable setting, we can go clean it up later if we need to and have everything connected for our mvp, I'm sure some of it is extra